packages/otelbin/src/app/s/[id]/route.ts

```diff
@@ -7,11 +7,7 @@ import { Redis } from "@upstash/redis";
 
 const redis = Redis.fromEnv();
 
-export async function GET(request: Request, { params }: { params: { id: string } }) {
+export async function GET(_: Request, { params }: { params: { id: string } }) {
 	const shortLink = await redis.get<string>(getShortLinkPersistenceKey(params.id));
-	return NextResponse.redirect(shortLink || "/", {
-		headers: {
-			"Cache-Control": "public, max-age=3600, stale-while-revalidate=3600, stale-if-error=3600",
```

Comment:
This can actually cause issues without the correct `Vary` response header. Hence caching is removed for this code path.